### PR TITLE
Add Collaboration Guidelines to Contribution Guide

### PR DESCRIPTION
**Pull request by doc-jones**
Include guidelines adopted from Mentored Sprints and The Recurse Center's Social Rules.  Some editorial freedoms applied.

### CODE REVIEW

File: book/src/contributing.md [hunk 1]
```diff
@@ -6,6 +6,64 @@ Want to help? Great! This is a fun project to hack on because it's relatively si
 
 We follow the [Rust code of conduct](https://www.rust-lang.org/policies/code-of-conduct). Please report any concerns to a [Lead][leads].
 
+## Optopodi Collaboration Guidelines
+
+Adopted from the [Mentored Sprints](https://www.mentored-sprints.dev) discord server and [The Recurse Social Rules](https://www.recurse.com/social-rules) these guidelines extend the [Rust CoC](https://www.rust-lang.org/policies/code-of-conduct) in ways that are actionable and clear examples to follow for the Optopodi team.
+```
```
Comment (nikomatsakis) [inline book/src/contributing.md:12]: I think I would rather have this be a "bullet list" in markdown than a code block

File: book/src/contributing.md [hunk 2]
```diff
@@ -6,6 +6,64 @@ Want to help? Great! This is a fun project to hack on because it's relatively si
...
+>Bob: What’s the name of the string copy function?
+Alice: Strncpy.
+Eve: (injecting into an existing conversation) You should use strlcpy. It’s safer.
+
+*No subtle -isms*
+
+>Carol: Windows is hard to use.
+Bob: No way. Windows is so easy to use that even my mom can use it.
+
+**And finally, not every comment requires a response.**
```
Comment (nikomatsakis) [inline book/src/contributing.md:65]: +1

**Review by nikomatsakis — CHANGES_REQUESTED** [review]
Looks good, one minor nit

File: book/src/contributing.md [hunk 3]
```diff
@@ -6,6 +6,63 @@ Want to help? Great! This is a fun project to hack on because it's relatively si
...
 
+## Optopodi Collaboration Guidelines
+
+Adopted from the [Mentored Sprints](https://www.mentored-sprints.dev) discord server and [The Recurse Social Rules](https://www.recurse.com/social-rules) these guidelines extend the [Rust CoC](https://www.rust-lang.org/policies/code-of-conduct) in ways that are actionable and clear examples to follow for the Optopodi team.
+
+Examples of behaviours that contribute to creating a positive environment:
+
+* Being friendly and patient  
+* Using welcoming and inclusive language  
+* eing respectful of differing viewpoints and experiences
```
Comment (chazkiker2) [inline book/src/contributing.md:17]: SP: can we correct "eing" to "Being"?

Comment (doc-jones) [reply book/src/contributing.md]: Add a B

File: book/src/contributing.md [hunk 4]
```diff
@@ -6,6 +6,63 @@ Want to help? Great! This is a fun project to hack on because it's relatively si
...
+
+Examples of behaviours that contribute to creating a positive environment:
+
+* Being friendly and patient  
+* Using welcoming and inclusive language  
+* Being respectful of differing viewpoints and experiences  
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+* Examples of unacceptable behaviour by participants include:
```
Comment (nikomatsakis) [inline book/src/contributing.md:21]:
```suggestion

Examples of unacceptable behaviour by participants include:
```

**Review by nikomatsakis — CHANGES_REQUESTED** [review]

**Pull request merged**